saas/sdk/app/auth.py

```diff
@@ -293,6 +347,20 @@ async def login_for_access_token(cls, form_data: OAuth2PasswordRequestForm = Dep
                 detail="Incorrect username or password",
                 headers={"WWW-Authenticate": "Bearer"},
             )
+        # The user account has been disabled due to exceeding the limit for failed login attempts
+        elif user == 'locked':
+            raise HTTPException(
+                status_code=status.HTTP_401_UNAUTHORIZED,
+                detail="The user account is locked due to exceeding the limit for failed login attempts",
+                headers={"WWW-Authenticate": "Locked"},
+            )
+        # The user account has been already disabled due to exceeding the limit for failed login attempts
+        elif user == 'disabled' or user.disabled:
```

Comment:
`user` shouldn't be a string. The flag should be sufficient (assuming the issues above are addressed).